proto/accessgraph/v1alpha/access_graph_service.proto

```diff
@@ -130,13 +136,56 @@ message EventsStreamRequest {
   }
 }
 
+// EventsStreamV2Request is a request to send commands to the access graph.
+// This command is used to sync the access graph with the Teleport database state.
+message EventsStreamV2Request {
+  // operation contains the desired operation
+  oneof operation {
+    // sync is a command to sync the access graph with the Teleport database state.
+    // it's issued once Teleport finishes syncing all resources with the database.
+    SyncOperation sync = 1;
+    // upsert is a command to put a resource into the access graph or update it.
+    ResourceList upsert = 2;
+    // delete is a command to delete a resource from the access graph when it's deleted from Teleport.
+    ResourceHeaderList delete = 3;
+    // access_lists_members is a command to upsert access lists members.
+    // It's issued when a user is added to an access list or when their membership is updated.
+    // It's also issued when a user membership expires but the user remains in the access list.
+    // In that case, the state of the user reflects the expiration reason.
+    AccessListsMembers access_lists_members = 4;
+    // exclude_access_list_members is a command to exclude access lists members
+    // when they are manually removed from the access list.
+    // If their membership remains but it's expired, they are not excluded
+    // from the access list but they don't receive any access - in that case,
+    // the request is [access_lists_members].
+    ExcludeAccessListsMembers exclude_access_list_members = 5;
+  }
+}
+
 // SyncOperation is a command that Teleport sends to the access graph service
 // at the end of the sync process.
 message SyncOperation {}
 
 // EventsStreamResponse is the response from EventsStream.
 message EventsStreamResponse {}
 
+// EventsStreamV2Response is the response from EventsStreamV2.
+message EventsStreamV2Response {
+  // action is a action that should be performed by Teleport on behalf of the access graph service.
+  oneof action {
+    // event is a audit event that should be logged by Teleport.
+    AuditEvent event = 1;
+  }
+}
+
+// AuditEvent is an event that should be logged by Teleport.
+message AuditEvent {
+  oneof event {
+    // access_path_changed is an event that should be logged when the access path changes.
+    events.AccessPathChanged access_path_changed = 1;
```

Comment:
I would add more details here and maybe renaming it to be something different than audit event.
I think we should also handle the tag -> teleport conversion in teleport itself instead of having TAG sending the prepared audit event

Reply:
> I would add more details here 

What details?

> I think we should also handle the tag -> teleport conversion in teleport itself instead of having TAG sending the prepared audit event

I was also thinking about it, but then you end up with every audit event duplicated in two places.

Reply:
What do you mean by every audit event duplicated?

Reply:
If we want to create an audit even we still need the audit event definition. If we want to have a separate definition for Teleport <-> TAG then we need a second definition that will have the same fields. 
I'm trying to say, that converting the audit events between Teleport - TAG sounds like a good idea, but this will require to create multiple proto messages with the same fields.

Reply:
>  return/add more details in one than what we have in the other.

@tigrato What is the advantage of returning more details from TAG and not recording them in the audit log?

Reply:
For now we might not have, but imagine we are interested in collecting posthog metrics. It would become easier to do so if we had separate entities